src/App.js

```diff
@@ -1,8 +1,26 @@
 import React from 'react';
 import './App.scss';
 
+import Sum from './components/Sum';
+
 const App = () => (
-  <h1>React sum</h1>
+  <section className="addNumbers">
+    <h1 className="addNumbers__title">React sum</h1>
+    <Sum
+      a={2}
+      b={2}
+    />
```

Comment:
Not necessary to use many new lines here. Code is short enough.
```suggestion
    <Sum a={2} b={2} />
```